pkg/handlers/webhandler.go

```diff
@@ -20,10 +20,12 @@ import (
 )
 
 const (
-	WebappVersion = "master"
+	WebappVersion      = "master"
+	WTLocations        = "WALKTHROUGH_LOCATIONS"
+	WTLocationsDefault = "https://github.com/integr8ly/tutorial-web-app-walkthroughs#master"
```

Comment:
I guess this will need an update, because the webapp "components" version will be handled separately from integr8ly components (and thus no master but rather 1.1.1)

Reply:
Not sure I follow on this one. I think defaulting to master is fine. Then have the installer set the location in the CustomResource during installation. 

@tremes we probably need a follow on here to change the installer to set the correct url based on the release?

Reply:
Well I am confused with the defaulting to master. I thought it's not the case for webapp & walkthroughs (in other words webapp & walkthroughs wouldn't have/use master tag at all). I don't have any problem to have master as default. 

@maleck13 yes I think you're right wrt installer setting the url. I can take a look